src/core/bitop.d

```diff
@@ -201,12 +213,15 @@ version (AnyX86)
      * using the X86 SSE4 POPCNT instruction.
      * POPCNT is not available on all X86 CPUs.
      */
+    pragma(DigitalMars_intrinsic, "popcnt")
     ushort _popcnt( ushort x ) pure;
```

Comment:
Unrelated to this PR I guess, but why not mark ``popcnt`` as an intrinsic instead?  Why this bizarrely named intrinsic here?